docs/.vuepress/sidebar/en.ts

```diff
@@ -935,14 +935,34 @@ export const enSidebar = sidebar({
       text: "Running a Hyperchain",
       collapsible: true,
       children: [
+        {
+          text: "Overview",
+          link: "/zk-stack/running-a-hyperchain/"
+        },
         {
           text: "Locally",
-          link: "/zk-stack/running-a-hyperchain/locally.md"
+          collapsible: true,
+          children: [
+            {
+              text: "Local",
+              link: "/zk-stack/running-a-hyperchain/locally/local.html",
+            },
+            {
+              text: "In Production",
+              link: "/zk-stack/running-a-hyperchain/locally/production.md"
+            }
+          ]
         },
         {
-          text: "In Production",
-          link: "/zk-stack/running-a-hyperchain/production.md"
-        }
+          text: "Rollup As A Service",
```

Comment:
Should be spelled Rollup as a Service, and abbreviated as RaaS

Reply:
👍